src/QMCWaveFunctions/WaveFunctionComponent.h

```diff
@@ -165,11 +165,6 @@ struct WaveFunctionComponent: public QMCTraits
     Optimizable = optimizeit;
   }
 
-  inline void setIonDerivs(bool calcionderiv)
-  {
-    ionDerivs=calcionderiv;
```

Comment:
Do we still need ionDerivs in WaveFunctionComponent.h? If not, please remove it.
If there is any unused ionDerivs, please remove them.

Reply:
Done.